### ISSUE TITLE
Gpu support in CLI needed

### ISSUE THREAD

**Issue by dPeS** [twosigma/Cook#793]
Hi,

When I added `--gpus` option here [1]:

`cs submit --executor mesos --gpus 1 sleep 10`
works ok (task is finished on gpu host), but:

```
[centos@ip-192-168-1-20]$ sudo cs submit --executor mesos --gpus 0 sleep 1
Attempting to submit on KKK cluster...
Job submission failed on KKK:
{"jobs": [{"gpus": "(not (pos? 0))"}]}
Job submission failed on all of your configured clusters.
```
any clues?

[1] https://github.com/twosigma/Cook/blob/master/cli/cook/subcommands/submit.py#L226

**Comment by dposada** [comment]
Hey @dPeS, thanks for the issue. The error you're getting is due to the fact that we have a server-side schema validation that doesn't allow gpus to be non-positive:

https://github.com/twosigma/Cook/blob/master/scheduler/src/cook/mesos/api.clj#L250

Can you leave off the `--gpus` flag altogether for the case where you don't need gpus?

**Comment by dPeS** [comment]
Hi Daniel,

Sure I'm good here - that doesn't block me - just reporting.
Please remember to add `--gpus` option to `submit.py`

**Comment by dposada** [comment]
Sounds good. I will take a look at this. Glad to hear you're not blocked.

**Comment by dPeS** [comment]
@dposada thanks, so I'm good here!

**Comment by dposada** [comment]
Resolved by PR #794
